modules/health_quest/app/controllers/health_quest/v0/lighthouse_appointments_controller.rb

```diff
@@ -8,15 +8,11 @@ def index
       end
 
       def show
-        render json: {}
+        render json: factory.get(params[:id]).response[:body]
       end
 
       private
 
-      def appointment_params
-        params.permit(:id)
-      end
```

Comment:
Removing this method since the controller is not assigning attributes to a model and the `show` action accesses the only `:id` param directly.